plugins/scaffolder-backend/api-report.md

```diff
@@ -123,6 +124,7 @@ export const createFilesystemRenameAction: () => TemplateAction<any>;
 // @public (undocumented)
 export function createGithubActionsDispatchAction(options: {
   integrations: ScmIntegrationRegistry;
+  githubCredentialsProvider: GithubCredentialsProvider;
```

Comment:
What's your reasoning behind making these required here and in the catalog processors?

Reply:
We can make them optional. Let me address that.

Reply:
I could do it in all of the processors / actions that pass the integrations through. There was one place that I could not default it :(

I made a change to the scaffolder action inteface. I changed ScmIntegrationsRegistry to ScmIntegrations but maybe I should have used the ScmIntegrationsRegistry interface throughout, thoughts?